utils/post.js

```diff
@@ -85,7 +85,7 @@ const handleCreatePostTO = (
   return removeDuplicate;
 };
 
-const modifyPollPostObject = async (userId, item, isBlurredPost = true) => {
+const modifyPollPostObject = async (userId, item, isBlurredPost = true, anonymUserId) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `modifyPollPostObject` has a new parameter `anonymUserId` added at the end. However, it is added after a parameter with a default value (`isBlurredPost = true`). In JavaScript, parameters with default values should be placed at the end of the parameter list to avoid confusion and potential errors.
```diff
- const modifyPollPostObject = async (userId, item, isBlurredPost = true, anonymUserId) => {
+ const modifyPollPostObject = async (userId, item, anonymUserId, isBlurredPost = true) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->